controllers/datamovement_controller_test.go

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2022 Hewlett Packard Enterprise Development LP
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import (
+	"context"
+	"os"
+
+	nnfv1alpha1 "github.com/NearNodeFlash/nnf-sos/api/v1alpha1"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gstruct"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var _ = Describe("Data Movement Test" /*Ordered, (Ginkgo v2)*/, func() {
+	var dm *nnfv1alpha1.NnfDataMovement = nil
+	var srcPath, destPath string
+	var err error
+
+	// test is using `sleep 1` for datamovement, so add some padding
+	dmTestTimeout := 3
```

Comment:
Is this 3 seconds? I think ginkgo supports string parsing, `"3s"` that might make this more obvious. Or you can use a time duration like `3 * time.Second`

Reply:
Looking at my other comments, I think the only use case here is in the finish with success, which might me you don't need this variable at all.

Reply:
Agreed. I was trying to be consistent, but that's not needed here. I'll remove it and only use it for the success case.